### ISSUE TITLE
module 'lkmltools' has no attribute 'grapher'

### ISSUE THREAD

**Issue by zhenliu2012** [ww-tech/lookml-tools#6]
Hi,

I am getting error message: AttributeError: module 'lkmltools' has no attribute 'grapher' by running `lkmltools.grapher.lookml_grapher` as described on your Documentation. 

It looks like the module doesn't have grapher method.

**Comment by weightwatchers-carlanderson** [comment]
Hi @zhenliu2012 
How are you running this?

This file runs OK:
```
$$ more test_grapher.py
from lkmltools.grapher.lookml_grapher import LookMlGrapher
LookMlGrapher({})
print("done!")
$$ python test_grapher.py
done!
```
as well as running this
```
$$ python run_grapher.py --config config/grapher/config_grapher.json
2019-09-26 16:00:39,572 INFO lookml_grapher.py extract_graph_info: Processing test/grapher_lkml/some_model.model.lkml
2019-09-26 16:00:39,574 INFO lookml_grapher.py extract_graph_info: Processing test/grapher_lkml/orphan_view.view.lkml
2019-09-26 16:00:39,575 INFO lookml_grapher.py extract_graph_info: Processing test/grapher_lkml/some_view.view.lkml
2019-09-26 16:00:39,576 INFO lookml_grapher.py extract_graph_info: Processing test/grapher_lkml/some_explore.explore.lkml
2019-09-26 16:00:39,577 INFO lookml_grapher.py extract_graph_info: Processing test/grapher_lkml/some_other_view.view.lkml
2019-09-26 16:00:39,578 INFO lookml_grapher.py run: Setting the following options: {'g': <networkx.classes.digraph.DiGraph object at 0x127407fd0>, 'filename': 'graph.png', 'title': 'test/grapher_lkml/*.lkml as of 2019-09-26', 'node_size': 500, 'label_font_size': 18, 'text_angle': 30, 'image_width': 24, 'image_height': 16}
2019-09-26 16:00:40,375 INFO lookml_grapher.py plot_graph: Graph written to graph.png
```

**Comment by zhenliu2012** [comment]
Hi @weightwatchers-carlanderson ,
I installed this package using pip
then I opened up my jupyter notebook instance and try to import grapher 
but it threw me that error msg.
My workaround for this was to copy this repo `lkmltools` to my working directory and run it in command line:
`python path_to_grapher/run_grapher.py --config config_grapher_project_name.json`

**Comment by weightwatchers-carlanderson** [comment]
@zhenliu2012 OK, thanks for the info. I will test this out.

**Comment by zhenliu2012** [comment]
Thanks!

**Comment by weightwatchers-carlanderson** [comment]
@zhenliu2012 this issue is now fixed in 2.0.3. The code was not packaged up correctly for distribution in pypi.
<img width="1164" alt="Screen Shot 2019-09-28 at 2 40 27 PM" src="https://user-images.githubusercontent.com/39501603/65821000-f67eee80-e1fd-11e9-9cf2-32ca6a8facd4.png">
